GetStream/stream-video-flutter
# [51] Google SignIn does not work on Android

### 🎯 Goal

Closes #51

### 🛠 Implementation details

Add predefined `debug.keystore` and its fingerprints to Firebase.

### 🧪 Testing

- install and launch `dogfooding` app
- tap on `Login with Google`
- choose/add your `stream.io` account

**Expected**: User gets navigated to the Home Screen

### ☑️Contributor Checklist

#### General

### ☑️Reviewer Checklist
